### ISSUE TITLE
Failed to access the underlying source code for analysis

### ISSUE THREAD

**Issue by wsw70** [gruns/icecream#68]
I just tried to test the prefixing, from the `python` (3.8) REPL and got this:

```
C:\WINDOWS\system32>python
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import arrow
>>> from icecream import ic
>>> ic.configureOutput(prefix=arrow.now().isoformat)
>>> ic(1)
2021-03-31T19:35:38.700968+02:00Error: Failed to access the underlying source code for analysis. Was ic() invoked in an interpreter (e.g. python -i), a frozen application (e.g. packaged with PyInstaller), or did the underlying source code change during execution?
1
>>> ic.configureOutput(prefix='aaa')
>>> ic('a')
aaaError: Failed to access the underlying source code for analysis. Was ic() invoked in an interpreter (e.g. python -i), a frozen application (e.g. packaged with PyInstaller), or did the underlying source code change during execution?
'a'
```

The prefix is printed, then the error message, and then the value to print.

**Comment by alexmojaki** [comment]
It's not going to work in a REPL.

**Comment by alexmojaki** [comment]
I think the warning `Was ic() invoked in an interpreter (e.g. python -i)` is meant to address this, but I don't think it's clear. Python is always an interpreter and the REPL doesn't depend on `-i`.

This kind of confusion is very common for this (#17) and similar libraries, but when I made a proposal about it on python-ideas I got no replies: https://mail.python.org/archives/list/python-ideas@python.org/message/CMM5ZV7GJPAR7KLEECETZP3SNFLCIKHQ/
